www/src/components/sidebar-body.js

```diff
@@ -105,9 +105,15 @@ class SidebarBody extends React.Component {
                     key={section.links[title]}
                     css={{ marginBottom: options.blockMarginBottom / 2 }}
                   >
-                    <Link to={section.links[title]} css={linkStyle}>
-                      {changedTitle}
-                    </Link>
+                    {
+                      section.links[title][0] == `#` ?
+                        <a href={section.links[title]} css={linkStyle}>
```

Comment:
oh maybe this is why clicking the links aren't scrolling? I think just using `<Link>` here would solve it.